lib/sign_in_page.dart

```diff
@@ -9,29 +11,136 @@ class SignInPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: Column(
+      body: Stack(
         children: [
-          TextField(
-            controller: emailController,
-            decoration: InputDecoration(
-              labelText: "Email",
-            ),
+          Column(
```

Comment:
The render overflow bug will appear on most screens when the keyboard is open, but the fix is easy:  wrap this Column with a [`SingleChildScrollView`](https://api.flutter.dev/flutter/widgets/SingleChildScrollView-class.html).

Don't wrap the Stack with a scrollview since then it won't know where to place the mountains svg on the screen (you can test this yourself if you wanna see what happens).